tilequeue/query/postgres.py

```diff
@@ -231,10 +234,15 @@ def make_queries_generator(sources, template_path, reload_templates):
 def parse_source_data(queries_cfg):
     sources_cfg = queries_cfg['sources']
     sources = []
-    for source_name, source_data in sources_cfg.items():
-        template = source_data['template']
-        start_zoom = int(source_data.get('start_zoom', 0))
-        source = make_source(source_name, template, start_zoom)
+    for source_name, templates in sources_cfg.items():
+        template_specs = []
+        for template_data in templates:
+            template = template_data['template']
+            zoom_start = int(template_data.get('zoom_start', 0))
+            zoom_stop = int(template_data.get('zoom_stop', 25))
```

Comment:
This just needs to be a larger number, but you're right that it's better to be consistent. I updated that in a90837d.